frontend/public/learn/Interactive-UI-Elements.elm

```diff
@@ -132,36 +141,26 @@ pattern used by interactive UI elements in Elm:
   [box]: http://library.elm-lang.org/catalog/elm-lang-Elm/latest/Graphics-Input#checkbox
 
 ```haskell
-checkbox : Handle a -> (Bool -> a) -> Bool -> Element
-checkbox handle processingFunction checked = ...
+checkbox : (Bool -> Signal.Message) -> Bool -> Element
+checkbox handler checked = ...
 ```
 
-The three arguments work like this:
+The two arguments work like this:
 
-  1. Clicking on a checkbox generates an event, and the handle specifies which
-     `Input` these events should be sent to. The generated event is a boolean
+  1. Clicking on a checkbox generates an event, and the handler specifies which
+     `Channel` these events should be sent to. The generated event is a boolean
      value that represents what the user *wants* the checkbox to be. Clicking an
      unchecked box generates a `True` event, and clicking a checked box generates
      a `False` event.
 
-  2. A way to process each event before sending it along to the `Input`
-     specified by the handle. This processing function lets us add
-     extra information to the event. A common thing to add is an ID to indicate
-     which checkbox has been clicked, so when there are four check boxes
-     reporting to the same `Input` we can send events like `(True, 1)` to
-     indicate that the first box wants to be checked. Adding extra information
-     is not always necessary, as in our synced checkbox example, so a common
-     processing function is `id` which passes the boolean value to the
-     specified `Input` unmodified.
-
-  3. The third argument is the actual state of the checkbox: whether it is
+  2. The second argument is the actual state of the checkbox: whether it is
      checked or not. This means `checkbox` is a [pure function][pure]! Whether
      it is checked or not is an argument, so that information must live in the
      model, not in the view!
 
   [pure]: http://en.wikipedia.org/wiki/Pure_function
 
-Breaking the concept of a checkbox into an `Input`, a `Handle`, and a `checkbox`
+Breaking the concept of a checkbox into a `Channel`, a `Signal.Message` handler, and a `checkbox`
```

Comment:
These three items parallel the old three arguments. Rework to match the new type signature.